teams/Aragon One/2019.md

```diff
@@ -73,27 +70,30 @@ Make the platform and core apps responsive so Aragon can be used from mobile bro
 
 [Related GitHub issue](https://github.com/aragon/design/issues/4)
 
-### I06 – Blockchain abstraction
+### ⏳ I06 – Blockchain abstraction
 
 The aim of this initiative is to lower the barrier of entry for adoption by allowing new users to immediately access functionality and perform actions without having to pay transactions fees.
 
 Focus areas for this initiative:
 
-- 🔜 Meta-transactions & relayers
-- 🔜 Improving onboarding for organizations
+- Meta-transactions and relayers
+
+- Improving onboarding for organizations
+
+**Status**: Initial research was conducted, but work on relayers and further integration into the Aragon client has been removed from the 2019 scope.
 
 [Related GitHub issue](https://github.com/aragon/aragonOS/pull/442)
 
-### I07 – New voting architecture
+### I07 – Research Aragon's scalability
 
-Redesign the current voting architecture and experience to accommodate new voting mechanisms and support for existing standard tokens in Aragon Voting apps.
```

Comment:
Instead of just voting, we have expanded this to also include research on the Aragon Chain, which will be a hot topic for us during Q4.